docs/message/message-schema.yaml

```diff
@@ -46,10 +32,24 @@ definitions:
         type: boolean
       chat:
         type: string
-        ref: '#/components/chat'
+      clearedFor:
+        type: array
+        items:
+          type: object
+          properties:
+            user:
+              type: string  
       createdAt:
         type: string
         format: date-time
       updatedAt:
         type: string
         format: date-time
+  messageFullInfo:
+    type: object
+    allOf:
+      - $ref: '#/definitions/message'
+      - properties:
+          author:
+            type: string
+            ref: '#/components/user'
```

Comment:
Don't add refs to non-existent schemas, leave here just `type: string` if this field contains only id of the user, or add reference to the user schema without any additional `type: ...`